libs/wingc/src/jsify/snapshots/access_property_on_value_returned_from_collection.snap

```diff
@@ -16,15 +16,15 @@ source: libs/wingc/src/jsify/tests.rs
 ## inflight.$Closure1-1.js
 
 ```js
-module.exports = function({ $_s___hello___length }) {
+module.exports = function({ $__obj__key_______if__obj_key______undefined__throw_new_Error__Map_does_not_contain_key_____key______return_obj_key______s___hello___length }) {
```

Comment:
That's odd looking. 😅 If it works that's all good, though I don't think the compiler should be generating super long identifiers like this. Open an issue?